cdap-cli/src/main/java/io/cdap/cdap/cli/command/CreateNamespaceCommand.java

```diff
@@ -54,31 +54,27 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     String hiveDatabase = arguments.getOptional(ArgumentName.NAMESPACE_HIVE_DATABASE.toString(), null);
     String schedulerQueueName = arguments.getOptional(ArgumentName.NAMESPACE_SCHEDULER_QUEUENAME.toString(), null);
     String rootDir = arguments.getOptional(ArgumentName.NAMESPACE_ROOT_DIR.toString(), null);
-    String exploreAsPrinc = arguments.getOptional(ArgumentName.NAMESPACE_EXPLORE_AS_PRINCIPAL.toString(), "true");
-    Boolean exploreAsPrincipal = Boolean.valueOf(exploreAsPrinc);
 
     NamespaceMeta.Builder builder = new NamespaceMeta.Builder();
     builder.setName(name).setDescription(description).setPrincipal(principal).setGroupName(groupName)
       .setKeytabURI(keytabPath).setRootDirectory(rootDir).setHBaseNamespace(hbaseNamespace)
-      .setHiveDatabase(hiveDatabase).setSchedulerQueueName(schedulerQueueName)
-      .setExploreAsPrincipal(exploreAsPrincipal);
+      .setHiveDatabase(hiveDatabase).setSchedulerQueueName(schedulerQueueName);
     namespaceClient.create(builder.build());
-    output.println(String.format(SUCCESS_MSG, name));
+    output.printf((SUCCESS_MSG) + "%n", name);
```

Comment:
Don't use printf and concatenation in same expression. Move `%n` to the constant

Reply:
Done